src/chrome/playSound.js

```diff
@@ -0,0 +1,20 @@
+import { SOUND_URL, SOUND_REPEAT } from '../constants';
+
+// Get sound options and play if enabled
+
+export const getAndPlaySoundNotificationOption = (store, mainAudioPlayer) => {
```

Comment:
Since this isn't specific to chrome, I think it would be better in the top-level (the chrome directory is for things that rely on the chrome API).